test/FunctionalTests/Microsoft.OData.Core.Tests/Evaluation/TestModel.cs

```diff
@@ -19,8 +19,10 @@ internal class TestModel
         public IEdmEntityType ProductType { get; set; }
         public IEdmEntityType DerivedProductType { get; set; }
         public IEdmEntityType MultipleKeyType { get; set; }
+        public IEdmEntityType DerivedMleMultiKeyType { get; set; }

```

Comment:
**Note:** The explanation for the refactor of the code in this test class in provided in the PR description